data/crac/crac-impl/src/main/java/com/farao_community/farao/data/crac_impl/remedial_action/network_action/ComplexNetworkAction.java

```diff
@@ -26,7 +26,8 @@
  * @author Viktor Terrier {@literal <viktor.terrier at rte-france.com>}
  */
 @JsonTypeName("complex-network-action")
-public class ComplexNetworkAction extends AbstractRemedialAction<NetworkAction> implements NetworkAction {
+@JsonSerialize(using = ComplexNetworkActionSerializer.class)
```

Comment:
I guess we should at this on all the Remedial Action classes of crac-impl ? (HvdcSetPoint ...)

Reply:
ok